Updated

sayem314 opened the pull request:
- Removed deprecated module request and replaced with axios
- Only keeps fully downloaded binary
- Refactored code with default prettier rules
- Performance improvements for binary lookup

sayem314 committed:
update dev dependencies

sayem314 committed:
replace request with axios

sayem314 committed:
refactor

TheBITLINK commented:
Can you configure prettier to use the [standard style](https://standardjs.com/) instead?

Everything else looks fine

sayem314 committed:
use standard.js styles

sayem314 committed:
basic types

sayem314 commented:
Since this module is not updated often I have added standard linter on scripts to manually set styles instead of installing multiple dev modules. Also added basic typescript declaration support to make it easier for people working with typescript.

sayem314 commented:
Can you please publish the latest updates?
